torch/testing/__init__.py

```diff
@@ -126,8 +136,11 @@ def _compare_tensors_internal(a: torch.Tensor, b: torch.Tensor, *, rtol, atol, e
 
         return (True, None)
 
+    if equal_nan == "relaxed":
```

Comment:
Is this necessary? The later uses of equal_nan cast it to bool.

Reply:
fixed